tests/unit/graph_reader/expected/module_import_all

```diff
@@ -0,0 +1,28 @@
+import copy
+
+import pandas
+
+
+def get_df():
+    df = pandas.DataFrame({"a": [1, 2]})
+    return df
+
+
+def get_df2(df):
+    df2 = pandas.concat([df, df])
+    return df2
+
+
+def run_all():
+    # Multiple return variables detected, need to save the variable
+    # right after calculation in case of mutation downstream
+    artifacts = []
+    df = get_df()
+    artifacts.append(copy.deepcopy(df))
```

Comment:
only concern here is that deepcopy might not work with all object types and this code generation might restrict us in the future.